build.gradle

```diff
@@ -47,6 +47,10 @@ java {
     // Workaround needed for Eclipse, probably because of https://github.com/gradle/gradle/issues/16922
     // Should be removed as soon as Gradle 7.0.1 is released ( https://github.com/gradle/gradle/issues/16922#issuecomment-828217060 )
     modularity.inferModulePath.set(false)
+
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(20)
```

Comment:
That needs to be 21

Reply:
I thought,, 20 is the latest release version (21 is in ramp down, isn't it? - https://openjdk.org/projects/jdk/21/). That version is used generally (tests, ...). Exception; When using jlink, 21 is used. I thought, it is a good idea.to modify one workflow only instead of changing all (JDK custom build download etc)

Reply:
I change this with sed in deployment.yml